projects/novo-elements/src/elements/chips/Chips.ts

```diff
@@ -273,7 +273,17 @@ export class NovoChipsElement implements OnInit, ControlValueAccessor {
   add(event) {
     if (event && !(event instanceof Event)) {
       this.items.push(event);
-      this.value = this.items.map((i) => i.value);
+      if (this.items[0].hasOwnProperty('category')) {
+        this.value = this.items.reduce((acc, val, i) => {
+          if (i) {
+            acc += ',';
+          }
+          acc += `${val.category.value}.${val.item.value}`;
+          return acc;
```

Comment:
this sounds like a very specific code change to solve one problem. can you please elaborate on what this is for?